### PR TITLE
Add paging

### PR DESCRIPTION
**Pull request by ovpoddar**

### CODE REVIEW

File: Shop/Controllers/ProductController.cs [hunk 1]
```diff
@@ -18,18 +18,25 @@ public IActionResult Index()
...
-            return View(model);
+            var i = PageNumber + 1;
+            switch (categoryId)
+            {
+                case 0:
+                    {
+                        var model = _productManager.GetModel(i);
+                        return View(model);
+                    }
+
```
Comment (cirrus42) [inline Shop/Controllers/ProductController.cs]: no space here please

Comment (ovpoddar) [reply Shop/Controllers/ProductController.cs]: ok

Comment (ovpoddar) [reply Shop/Controllers/ProductController.cs]: I'll fix it soon

File: Shop/Views/Product/Index.cshtml [hunk 2]
```diff
@@ -3,36 +3,42 @@
...
+    <form method="post" id="form">
+        <table>
+            <thead>
+                <tr>
+                    <td>Product Name</td>
+                    <td>Price</td>
+                    <td>Brand</td>
+                    <td>
+                        <select onchange="document.getElementById('form').submit()" name="categoryId">
+                            <option>Categories</option>
```
Comment (ovpoddar) [inline Shop/Views/Product/Index.cshtml]: can i use java script to the value for categoryId??

**Comment by ovpoddar** [comment]
paging compleate

**Pull request merged**